spec/models/department_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe Department, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
For other models I've used this spec as a place to list all attributes it should respond to. I think that's a nice convention; what do you think?